src/app/meetups/meetups-add.component.html

```diff
@@ -0,0 +1,14 @@
+<form class="form-horizontal" (ngSubmit)="onSubmit(meetupForm.value)" #meetupForm="ngForm">
+    <legend>Start a New Meetup</legend>
+    <div class="fields-container">
+      <mat-form-field>
+        <input matInput name="title" type="text" ngModel i18n-placeholder placeholder="Title" required=""  >
```

Comment:
There is no end of the tag input. Could you please add the end close of input tag ? And also required attribute is empty . It needs to be either true or false.

Reply:
Could you also please check Travis fail ?

Reply:
still working on meetup form ( work in progress ) , added to TODO list. Thanks for feedback